### PR TITLE
feat(ruby): Switch to @renovate/ruby-semver library

### PR DESCRIPTION
**Pull request by zharinov**

### CODE REVIEW

**Comment by ViceIce** [comment]
@zharinov @rarkins  Is there any reference why we want to do this?

**Comment by ViceIce** [comment]
can we extend the readme for https://github.com/renovatebot/ruby-semver before merging this? so it is more clear why we have our own package. Also other projects can reuse when they like.

I we don't want to let thos libs be reusable, we should make renovate a monorepo without publishing those libs to the wild.

**Comment by rarkins** [comment]
> I we don't want to let thos libs be reusable, we should make renovate a monorepo without publishing those libs to the wild.

Because of licensing differences, it's easiest to keep ported version schemes in separate repositories instead of mixing license types within this one.

File: package.json [hunk 1]
```diff
@@ -96,8 +96,8 @@
   },
   "dependencies": {
     "@renovate/pep440": "0.4.1",
+    "@renovatebot/ruby-semver": "^0.1.0",
```
Comment (rarkins) [inline package.json]: Please change to 0.1.2

**Review by viceice — APPROVED** [review]

**Pull request merged**